src/sections/Projects/Sistent/components/list/code.js

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import { navigate } from "gatsby";
+import { useLocation } from "@reach/router";
+import { List, ListItemIcon, ListItemAvatar, ListItemText,ListItem,ListSubheader, ListItemButton, SistentThemeProvider, Divider } from "@layer5/sistent";
+import { CodeBlock } from "./code-block";
+import { SistentLayout } from "../../sistent-layout";
+import TabButton from "../../../../../reusecore/Button";
+import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
+
+const codes = [
+  // Basic List with List Items
+  `  <List>
+         <ListItem><ListItemText primary="Layer5 Sistent 1" /></ListItem> 
+         <ListItem><ListItemText primary="Layer5 Sistent 2" /></ListItem> 
+         <ListItem><ListItemText primary="Layer5 Sistent 3" /></ListItem> 
+  </List> `,
+  // List with Icons in List Items
+  `  <List>
+        <ListItem><ListItemIcon>🌟</ListItemIcon><ListItemText primary="Layer5 Sistent Starred" /></ListItem>
+        <ListItem><ListItemIcon>📅</ListItemIcon><ListItemText primary="Layer5 Sistent Calender" /></ListItem>
+        <ListItem><ListItemIcon>🔔</ListItemIcon><ListItemText primary="Layer5 Sistent Notification" /></ListItem>
+  </List> `,
+  // List with Avatars
+  `  <List>
+        <ListItem><ListItemAvatar>👤</ListItemAvatar><ListItemText primary="Layer5 Sistent User 1" /></ListItem>
+        <ListItem><ListItemAvatar>👩‍💻</ListItemAvatar><ListItemText primary="Layer5 Sistent User 2" /></ListItem>
+        <ListItem><ListItemAvatar>👤</ListItemAvatar><ListItemText primary="Layer5 Sistent User 3" /></ListItem>
+  </List> `,
+  // List with Subheader
+  `  <List>
+        <ListSubheader>Section 1</ListSubheader>
+        <ListItem><ListItemText primary="Layer5 Sistent A" /></ListItem>
+        <ListItem><ListItemText primary="Layer5 Sistent B" /></ListItem>      
+        <ListSubheader>Section 2</ListSubheader>
+        <ListItem><ListItemText primary="Layer5 Sistent C" /></ListItem>
+        <ListItem><ListItemText primary="Layer5 Sistent D" /></ListItem>
+  </List> `,
+  // List with Action Buttons
+  `  <List>
+        <ListItemButton onClick={() => alert("Clicked!")}> Layer5 Sistent Action 1</ListItemButton>
```

Comment:
@csengineer1990 If you could provide a nested list code example, here is example [link](https://v5.mui.com/material-ui/react-list/#nested-list)

Reply:
Should I add the nested list code example only in the code.js file, or should I also include it in the overview and guidance section to explain the concept of nested lists?